OpenUpSA/wazimap-ng-ui
# Downloading the map

## Description
Merging the latest design changes and downloading the current state of the map through a download button

## Related Issue
https://github.com/OpenUpSA/wazimap-ng-ui/issues/104

## How to test it locally
* Run the project
* Click the download button 
* Check if the exported image matches the current map

## Screenshots
Here is a sample exported image

![image](https://user-images.githubusercontent.com/53019884/97905488-224e9b00-1d53-11eb-8ac5-9e013bcab41a.png)


## Changelog

### Added
* `map > map_download.js` has the code to export image of the map. Exported image is created using `html2canvas` plugin.

### Updated
* moved `saveAs()` function from `reusable-charts > horizontal-bar-chart.js` to `utils.js` so it can be used from `map > map_download.js` too

### Removed


## Checklist


### Pull Request


### Commits


### Code Quality


### Testing
